process_battery_data.py

```diff
@@ -0,0 +1,23 @@
+from battery_limits import bms_range_limits
+     
+def process_data(bms_attributes):
+    info = []
+    for param in bms_attributes.items():
```

Comment:
bms_attributes is a dictionary, is that right?
then you can do
`for param_name in bms_attributes:` to iterate through its keys.

Reply:
I wanted the key as well as value pair that is why the items(). And i did not want to write another line passing the key to get the value.